_troubleshoot/index.md

```diff
@@ -116,3 +116,10 @@ The security plugin blocks the update by script operation (`POST <index>/_update
 ## Illegal reflective access operation in logs
 
 This is a known issue with Performance Analyzer that shouldn't affect functionality.
+
+
+## bootstrap checks failed: max virtual memory areas vm.max_map_count is too low
+
+To increase memory and make a change permanent
+edit /etc/sysctl.conf and set vm.max_map_count to 262144
+you can check it using sysctl vm.max_map_count
```

Comment:
## Vm.max_map_count is too low

If you're using a production workload, be sure to set the [Linux setting](https://www.kernel.org/doc/Documentation/sysctl/vm.txt) `vm.max_map_count` to 262144. See [Important settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings) for more information.